src/Build/Instance/ProjectInstance.cs

```diff
@@ -564,8 +564,7 @@ private ProjectInstance(ProjectInstance that, bool isImmutable, RequestedProject
             _hostServices = that._hostServices;
             _isImmutable = isImmutable;
             _evaluationId = that.EvaluationId;
-
-            TranslateEntireState = that.TranslateEntireState;
+            _translateEntireState = that._translateEntireState;
```

Comment:
There is a conditional `_translateEntireState = false` further down in this constructor. Have you verified that it cannot break the scenario of building solution/meta-projects?

Reply:
That conditional `_translateEntireState = false` is not used by traversal project items (such as sln). I have verified it with Rainer, and it shall be OK.